### PR TITLE
Remove mentions of Carmine in this repository

### PR DESCRIPTION
**Pull request by tensojka**
The instance of Konoha governing the Carmine Options AMM can be found at https://github.com/CarmineOptions/amm-governance

### CODE REVIEW

File: src/treasury_types/carmine.cairo
Comment (Chepelau) [inline src/treasury_types/carmine.cairo:1]: What is this file here for? It's carmine specific no?

File: Scarb.toml [hunk 1]
```diff
@@ -1,15 +1,18 @@
 [package]
 name = "konoha"
 description = "A flexible monolithic governance contract, developed for use with Carmine Options AMM"
-version = "0.3.0"
+version = "0.4.0"
 cairo-version = "2.6.3"
 
 [dependencies]
 cubit = { git = "https://github.com/influenceth/cubit.git", commit = "62756082bf2555d7ab25c69d9c7bc30574ff1ce8" }
```
Comment (Chepelau) [inline Scarb.toml:8]: Is cubit dependency needed for the governance?

File: src/treasury_types/carmine.cairo
Comment (tensojka) [reply src/treasury_types/carmine.cairo:1]: The treasury can deposit into Carmine liquidity pools. In the future, we want to be able to deposit to Nostra/ZkLend too, which is why I moved it to treasury_types directory

File: Scarb.toml (hunk 1 above)
Comment (tensojka) [reply Scarb.toml]: No longer, removed, thanks

File: Scarb.toml [hunk 2]
```diff
@@ -1,15 +1,17 @@
 [package]
 name = "konoha"
 description = "A flexible monolithic governance contract, developed for use with Carmine Options AMM"
```
Comment (Chepelau) [inline Scarb.toml:3]: The contract is now for general use, not "for use with Carmine Options AMM".

**Pull request merged**